gcsio/src/main/java/com/google/cloud/hadoop/gcsio/CreateFileOptions.java

```diff
@@ -44,9 +44,9 @@ public enum WriteMode {
   public static Builder builder() {
     return new AutoValue_CreateFileOptions.Builder()
         .setAttributes(ImmutableMap.of())
-        .setContentType(CreateObjectOptions.CONTENT_TYPE_DEFAULT)
+        .setContentType(CreateObjectOptions.DEFAULT_OVERWRITE.getContentType())
         .setEnsureNoDirectoryConflict(true)
-        .setMinSyncInterval(Duration.ofSeconds(10))
+        .setMinSyncInterval(Duration.ZERO)
```

Comment:
why changing the default of 10 seconds?

Reply:
For consistency between GCS connector and GCSIO default settings.